mobius-rx/src/test/java/com/spotify/mobius/rx/RxEventSourcesTest.java

```diff
@@ -70,4 +79,65 @@ public void errorsAreForwardedToErrorHandler() throws Exception {
 
     rxErrorsRule.assertSingleErrorWithMessage("crash!");
   }
+
+  @Test
+  public void noEventsAreGeneratedAfterDispose() throws InterruptedException, ExecutionException {
+    // given an observable that continuously emits stuff
+    ExecutorService executor = Executors.newSingleThreadExecutor();
+    PublishSubject<Integer> subject = PublishSubject.create();
+
+    final AtomicInteger count = new AtomicInteger();
+    final AtomicBoolean stop = new AtomicBoolean();
+
+    Future<?> future =
+        executor.submit(
+            () -> {
+              while (!stop.get()) {
+                subject.onNext(count.incrementAndGet());
+              }
+            });
+
+    // and an event source based on that observable
+    EventSource<Integer> source = RxEventSources.fromObservables(subject);
+
+    // when an event source is created (many times to make this non-flaky/less flaky)
```

Comment:
Did you meant to write "_when an event **consumer** is created_" here? (same for line 111 and the others tests as well)

Reply:
i think it is is meant to say something like "when an event source is subscribed to" here, and below "the event consumer doesn't receive events after it has been disposed"?

Reply:
Good points, fixed!